sql/src/test/java/io/crate/analyze/symbol/FunctionTest.java

```diff
@@ -35,6 +35,7 @@
 import java.util.EnumSet;
 import java.util.Set;
 
+import static com.carrotsearch.randomizedtesting.RandomizedTest.randomAsciiOfLength;
```

Comment:
This import is now necessary although it might look like it has been added accidentally.

`randomAsciiOfLength` was a function in `ESTestCase` which called `RandomizedTest.randomAsciiOfLength`. This function has been removed in ES 5.4.3, hence the direct import in this subclass.